settings.gradle.kts

```diff
@@ -22,6 +22,7 @@ rootProject.name = "collector"
 plugins {
     // Required to publish Gradle Build Scans from Github Workflow
     id("com.gradle.enterprise") version("3.9")
+    id("org.gradle.toolchains.foojay-resolver-convention") version("0.7.0")
```

Comment:
Would be good to add a reason why this plugin is necessary, you wrote in the commit:

> Add toolchain repository to make the build Gradle 8.0 compatible.